QUEST.tsv

```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
+QUEST_20150714_002073	$Even Lydia Schaffen, friend of Ruklys, turned to the side of kingdom soldiers. {nl}No one will stand on your side. {nl}With your hands, collapse The Great Earth Fortress, the proof of your treason, and wave the white flag. {nl}Faithful Premier Eminent shall forgive all.
+QUEST_20150714_002074	$Are you going to die with Ruklys who misled you with malicious tricks to create The Great Earth Fortress to stand against the kingdom? {nl}Or live a new life by serving the king until the end by following the way of Premier Eminent who protected the kingdom from the rebellion?
+QUEST_20150714_002075	$The Records of Rebel Forces
+QUEST_20150714_002076	$As I was retreating from kingdom soldiers, I encountered many demons. {nl}It doesn't make sense. Where did they come from?
+QUEST_20150714_002077	$There's no hope at all. Ah… my lovely Ridell. {nl}The kingdom soldiers did not keep their word. We're all going to die.
+QUEST_20150714_002078	$Those kingdom soldiers! Especially that Premier Eminent! They're no different from demons. {nl}Even the Goddesses would not forgive you.
+QUEST_20150714_002079	$I don't want to die. {nl}We were just following the the Goddesses' words. {nl}Everyone is killing each other while in the names of the Goddesses. {nl}Who has deceived to us? {nl}
+QUEST_20150714_002080	$Kingdom Military Recruit Propaganda
+QUEST_20150714_002081	$Kingdom Military needs competent people like you! {nl}Loyalty to the kingdom! To victory and a brighter future!
+QUEST_20150714_002082	$Swear loyalty to Kingdom Military and become a new person. {nl}The Goddesses protect our armies.
+QUEST_20150714_002083	$Life and death… which one would you choose? {nl}The only way to survive is to swear loyalty to the kingdom!
+QUEST_20150714_002084	$Don't forget that we won 600 years ago. {nl}Join the military. You sins can be forgiven.
+QUEST_20150714_002085	$Please move with us if possible. {nl}There's been a number of monster sightings near here.
+QUEST_20150714_002086	$The supplies should be here soon… {nl}Are they going to be late again?
+QUEST_20150714_002087	$You will need various tools when travelling. {nl}It's better to buy them in advance.
+QUEST_20150714_002088	$Goddess Laima… Actually, I don't know her that well. No one does, except for the disciples. {nl}I heard she's the Goddess of Destiny and Forecasts, but she has never shown up.
+QUEST_20150714_002089	$I just remembered something while I was talking to you. {nl}Maybe Goddess Laima knew everything about the disaster beforehand.
+QUEST_20150714_002090	$I pray to Goddess Gabija everyday. It keeps the house warm. {nl}As you may already know, Goddess Gabija controls fire and since I'm a housewife, I am close to fire as well.
+QUEST_20150714_002091	$If you cleverly use the dry air that forms around severe colds, you'd be able to freeze any object even in a desert. {nl}However, if you can't receive the blessing from the Goddess, it won't be possible to make that kind of air.
```

Comment:
change line 1191 to this! seems to be the same, but doesn't have the issue you introduced through your edit in line 1191